doc/api/http2.md

```diff
@@ -2047,7 +2047,8 @@ changes:
     queued to be sent, and unacknowledged `PING` and `SETTINGS` frames are all
     counted towards the current limit. **Default:** `10`.
   * `maxHeaderListPairs` {number} Sets the maximum number of header entries.
-    The minimum value is `4`. **Default:** `128`.
+    This is similar to `HTTP/1`'s `maxHeadersCount`. The minimum value is `4`.
```

Comment:
```suggestion
    This is similar to `http.Server#maxHeadersCount`. The minimum value is `4`.
```

and add a link to the bottom. same as below

```md
[`http.Server#maxHeadersCount`]: http.html#http_server_maxheaderscount
```

Reply:
see
https://github.com/nodejs/node/blob/d7b02c3cadb9f725579df30aa6bb082399961fbf/doc/api/https.md#servermaxheaderscount

Reply:
I actually considered this change before making the pull request, but `http.ClientRequest` also has a similar property and I think explicitly specifying `http.Server` would indicate that it's different from the former, i.e., they both seem to differ in functionality.

I think we should add something like "This is similar to `http.Server#maxHeadersCount` or `http.ClientRequest#maxHeadersCount`", *if* we want to make this change. What do you think?

Reply:
I haven’t any special opinions on this. But this should get more discussion with other collaborators.

Reply:
I would go with the suggestion made by @Himself65. Just naming one seems sufficient as an example reference while also being fine with naming both.

@Trott do you have any opinion on this?

Reply:
ping @Trott

Reply:
> ping @Trott

I prefer adding both for the reason suggested by @rexagod.